{{ cookiecutter.project_name }}/.github/workflows/update.yaml

```diff
@@ -0,0 +1,55 @@
+---
+name: Update project structure
+
+on:
+  schedule:
+    - cron: "0 2 * * 1" # Every Monday at 2am
```

Comment:
oh cool did not know about dispatch jobs. Here is the issue, for a dispatch job to work we would have to know the target repos, that means everytime I create a repo outta this template I will have to update the template telling the new repos link as well so that when any updates to template logic or structure are made, a dispatch event is triggered, this is less than ideal.

Reply:
Okay, fair. Perhaps sth we can look into later. Do you want to create an issue?

Reply:
do we need an issue? cruft suggests it to be done [this way](https://cruft.github.io/cruft/#:~:text=always%20submitted%20upstream.-,Automating%20updates%20with%20GitHub%20Actions,-If%20you%20have) though, maybe you have another thing in mind?